app/src/main/java/com/yeonae/chamelezone/view/home/adapter/HomePlaceRvAdapter.kt

```diff
@@ -5,9 +5,9 @@ import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import androidx.recyclerview.widget.RecyclerView
-import com.yeonae.chamelezone.view.place.PlaceDetailActivity
 import com.yeonae.chamelezone.R
 import com.yeonae.chamelezone.data.model.Place
+import com.yeonae.chamelezone.view.place.PlaceDetailActivity
 import kotlinx.android.synthetic.main.item_place_list.view.*
 
 class HomePlaceRvAdapter(var placeList: ArrayList<Place>) :
```

Comment:
```suggestion
class HomePlaceRvAdapter(private val placeList: ArrayList<Place>) :
```
var 로 선언할 필요가 없을 것 같아요.
또한 private 으로 외부에서 adapter의 아이템 데이터에 접근하지 못하도록 제한해주는 것이 좋아보입니다.

Reply:
수정했습니다!
d15a5aa